bundle/config/mutator/populate_current_user.go

```diff
@@ -18,6 +18,10 @@ func (m *populateCurrentUser) Name() string {
 }
 
 func (m *populateCurrentUser) Apply(ctx context.Context, b *bundle.Bundle) error {
+	if b.Config.Workspace.CurrentUser != nil {
+		return nil
+	}
```

Comment:
Curious: has this run twice in your testing?

Reply:
Yes, it would, but I _think_ that may have only happened with `databricks deploy --run`. So I'll remove this.

We should actually have a general fix to make sure `w.CurrentUser.Me()` is cached, since it is very very slow and was still being called a second time in some other cases. This could be part of a general initiative to make things a bit faster.